docs/api-site-config.md

```diff
@@ -143,6 +143,33 @@ h1 {
 * `defaultLang` defines a default language. It will be used if one is not specified at the top of the code block. You can find the [list of supported languages here](https://github.com/isagalaev/highlight.js/tree/master/src/languages).
 * `themeUrl` is the custom URL of CSS theme file that you want to use with Highlight.js. If this is provided, the `theme` and `version` fields will be ignored.
 
+`layouts` - You can specify some custom components here to override the [offcial layout template](https://github.com/facebook/Docusaurus/blob/master/lib/core/Doc.js) to render some or all of the doc pages. 
+
+```js
+layouts: {
+  // default will be applied to all doc pages
+  default: function({ React, MarkdownBlock }) {
```

Comment:
I don't think is true. From reading the code, I don't think we have a way now to specify a default layout for all pages. This is simply a layout called `default` without any special functionality.

Reply:
https://github.com/facebook/Docusaurus/pull/763/files#diff-fe37ce87a7b7b83106303bf9233b49fbR238

Reply:
My bad, I missed out on that chunk. I was looking at existing code as I had the impression this PR was only documenting the layouts behavior.

Reply:
Actually i believe PR #128 is incomplete. It is only available at dev server, not on build. Might need to modify

https://github.com/facebook/Docusaurus/blob/c2ff41365039c2d86f9d5d6da0370cae31ab0a32/lib/server/generate.js#L195

Reply:
@endiliey Added.

Reply:
Good catch @endiliey! We really need to reduce the double work in server.js and generate.js